internal_servers/orthanc_reciever_agent.py

```diff
@@ -125,11 +134,38 @@ def process_data(brains_tool_path: str, study_id: str, input_data_path: str, out
         print(f"Error executing script: {e}")
 
 
-def send_data_to_hospital(data_path, hospital_id):
+def upload_data_to_internal(directory_path: str, orthanc: pyorthanc.Orthanc):
```

Comment:
What is the difference between upload_data_to_internal() and return_to_original_PACS()? What is each used for?

Reply:
I will provide a bit longer response to make sure there is enough context.

!!! IMPORTANT: only PACS system can communicate with PACS system !!!

This is the simplified workflow:
1. Hospital PACS sends data to our internal PACS
2. Internal downloads data to local system
3. data is processed
4. new data is uploaded back to the Internal PACS
5. Internal PACS returns data back to the Hospital PACS

We cannot return the data directly to the hospital. We first have to upload the results to the internal PACS and then only the PACS can send data back to the Hospital PACS

Let me know if there are other questions about that.